folium/folium.py

```diff
@@ -440,6 +158,10 @@ def line(self, locations,
             or a tuple of the form (Vincent object, 'vis_path.json')
             It is possible to adjust the width of text/HTML popups
             using the optional keywords `popup_width` (default is 300px).
+        latlon: bool, default True
+            Whether locations are given in the form [[lat,lon]] or not ([[lon,lat]] if False).
+            Note that the default GeoJson format is latlon=False,
+            while Leaflet polyline's default is latlon=True.
```

Comment:
My goal in `folium/folium.py` was not to break the API to ensure backward compatibility as much a s possible. A new API would be designed progressively in `folium/map.py`